src/libs/actions/Session/index.js

```diff
@@ -27,23 +24,6 @@ Onyx.connect({
     callback: val => credentials = val,
 });
 
-/**
- * Sets API data in the store when we make a successful "Authenticate"/"CreateLogin" request
- *
- * @param {Object} data
- * @param {String} data.accountID
- * @param {String} data.authToken
- * @param {String} data.email
- */
-function setSuccessfulSignInData(data) {
-    PushNotification.register(data.accountID);
```

Comment:
Wait, this can't be removed, it needs to exist somewhere or push notifications don't work

Reply:
Should be fine, added a console log in to check

https://user-images.githubusercontent.com/16747822/208995081-363a63f8-5fc0-419e-819b-35007b103b65.mov

Reply:
So odd, so that call was useless? You sure it is not used in some other flow? Like signing in normally?

Reply:
Seems like it, runs on normal sign in as well

Reply:
Oh wow.... just to be sure, tagging @Julesssss, @arosiclair and @cristipaval that are probably familiar with this since they are working on the notifications doc.

Reply:
Yeah as long as the new accountID gets set on the session, the code @aimane-chnaif linked should register notifications.